hi-ml-multimodal/src/health_multimodal/vlp/inference_engine.py

```diff
@@ -27,6 +27,24 @@ def __init__(self,
         self.image_inference_engine = image_inference_engine
         self.text_inference_engine = text_inference_engine
 
+    def get_similarity_score_from_raw_data(self,
```

Comment:
I'm not sure what we mean by `from_raw_data` or why it's needed, but I guess it's ok if you want to keep it consistent with the other method.